differential-dataflow/src/operators/iterate.rs

```diff
@@ -169,7 +169,7 @@ where
 impl<G: Scope, D: Data, R: Abelian, C: Container + Clone + 'static> Variable<G, D, R, C>
 where
     G::Timestamp: Lattice,
-    StreamCore<G, C>: crate::operators::Negate<G, C> + ResultsIn<G, C>,
+    StreamCore<G, C>: crate::operators::Negate<C, G> + ResultsIn<G, C>,
```

Comment:
I think conventionally, we usually have scope parameters first and containers later (e.g. timely's `Map` trait, as the first one I found). I'm looking at my local DD repo, and it has `Negate<G, C>` in it, where `G: Scope`. I'm not sure I understand the flipping that is going on here.